### ISSUE TITLE
Compatibility with react-leaflet@1.X.X

### ISSUE THREAD

**Issue by danmfisher** [OpenGov/react-leaflet-marker-layer#4]
Get the following error when using marker layer with current version of react-leaflet / leaflet.
```
MapLayer.js:60 Uncaught Error: createLeafletElement() must be implemented
    at MarkerLayer.createLeafletElement (MapLayer.js:60)
    at MarkerLayer.componentWillMount (MapLayer.js:72)
```

react-leaflet@1.1.6
leaflet@1.0.3

Are there plans to upgrade this library to support these?

**Comment by MartinSahlen** [comment]
You should have a look at https://github.com/OpenGov/react-leaflet-heatmap-layer/pull/10 to get an understanding of where to look to support the new version of leaflet. It shouldnt be too much work. Most of the thoughtwork is done so should just be to copy-paste the relevant parts.

**Comment by danmfisher** [comment]
Yes, I follow the thoughtwork. I am wondering about the upgrades to the versions distributed by npm as I'd prefer to keep my deployment set up clean, and currently I install packages from npm inside my Docker container. Do you know if there are any plans to update the versions distributed by npm?

**Comment by MartinSahlen** [comment]
You could either fork it, and create a pull request in this repo. and then use your fork in your package.json until your prq gets merged by @jeremiahrhall in this repo. It is not up to the maintainer of this repo to publish new package versions to npm as well, after the change is made here. They have not been the fastest based on my experience with my experience with https://github.com/OpenGov/react-leaflet-heatmap-layer/pull/10, but one can hope 😁

**Comment by jeremiahrhall** [comment]
@danmfisher I've merged the react-leaflet 1.x PR and published a new version (1.0.0) of react-leaflet-heatmap-layer to npm. Please let me know if this resolves this issue for you.

**Comment by DoubleYo** [comment]
I fix this issue in PR #5
